src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -38,6 +40,9 @@ const ActionSidebar: FC<PropsWithChildren<ActionSidebarProps>> = ({
 }) => {
   const { action, defaultValues, loadingAction, isMotion, motionState } =
     useGetActionData(transactionId);
+  const { expenditure, loadingExpenditure } = useGetExpenditureData(
+    action?.expenditureId,
+  );
```

Comment:
This hook seems to be called in `useGetActionData` as well, so maybe whatever data you need here could be exported from that hook instead?